runhouse/resources/hardware/cluster.py

```diff
@@ -177,6 +172,38 @@ def save_config_to_cluster(
             node=node or "all",
         )
 
+    def update_cluster_config_on_cluster(
+        self, items_to_update: Union[dict, list[dict]]
```

Comment:
why does this method need to accept different input types? can't it always accept a dictionary, since the keys will be unique even if there are more than one item?

Reply:
for example - what if the method just accepts **kwargs, and then we call it with `update_cluster_config_on_cluster(den_status_ping_interval=-1)`